lib/fathead/electron_api/parse.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import requests
+import sys
+import re
+from bs4 import BeautifulSoup
+
+reload(sys)
+sys.setdefaultencoding('utf8')
+
+START_URL = 'http://electron.atom.io/docs/api/'
+OUTPUL_FILE = open('output.txt', 'w+')
+RESULT_URLS = []
+
+
+# Print the content to the output file
+def print_article_line(title, content):
+
+    content = content.replace('\n', '\\n').replace('\t', '  ')
+    abstract = '<section class="prog__container">' + content + '</section>'
+    abstract = ''.join([i if ord(i) < 128 else ' ' for i in abstract])
+
+    list_of_data = [
+        title,      # 1. article title
+        'A',        # 2.type is article
+        '',         # 3.no redirect data
+        '',         # 4.leave empty
+        '',         # 5.no categories
+        '',         # 6.leave empty
+        '',         # 7.no related topics
+        '',         # 8.leave empty
+        '',         # 9.an external link back to home
+        '',         # 10.no disambiguation
+        '',         # 11.images
+        abstract,   # 12.abstract
+        START_URL   # 13.url to doc
+    ]
+
+    OUTPUL_FILE.write('{}\n'.format('\t'.join(list_of_data)))
+
+
+# Extracts content of all the pages
+def get_func_desc():
+
+    for url in RESULT_URLS:
+        title = url.split('/')[-1].lower().replace('-', ' ')
+        content = ""
+
+        page = requests.get(url)
+        src = page.text
+
+        idx = src.find('<h2')
+        main_desc = src[:idx]  # To extract the main content from a page
+        ob_main = BeautifulSoup(main_desc, 'html.parser')
+
+        info = ob_main.find('blockquote')
+
+        while info:
+            if info.name == 'p':
+                content = content + info.text + '\n\n'
+            elif info.name == 'ul':
```

Comment:
Please add `class="prog__ul"` to the ul. This will allow it to show bullet points, once I have deployed the CSS changes I'm working on.

The result will eventually look much better:
<img width="1428" alt="cursor_and_electron_systempreferences_getcolor___windows__javascript__at_duckduckgo" src="https://cloud.githubusercontent.com/assets/873785/22480257/c89bfae4-e7be-11e6-9a37-1c9a7bc3a7bb.png">